docs/troubleshoot/troubleshoot-zos-certificate.md

```diff
@@ -25,12 +25,57 @@ Dkeystore.pkcs12.keyProtectionAlgorithm=PBEWithSHAAnd3KeyTripleDES -J-
 Dkeystore.pkcs12.keyPbeIterationCount=50000
 ```
 
-- Set the flag `keystore.pkcs12.legacy` enabled with no value to create a PKCS12 keystore that can be loaded.
+- Set the flag `keystore.pkcs12.legacy` to enabled with no value to create a PKCS12 keystore that can be loaded.
 
-**Notes**
+:::note
+* If you already have an existing keystore or you are using keyrings, this error will not happen.
+* If you do not use ZSS, this error will not happen. ZSS is enabled by default.
+* If you already use your own PKCS12 files instead of the files that Zowe generates for you, this error will not happen. 
+:::
 
-If you already have an existing keystore or you are using keyrings, this error will not happen.
+## Eureka request failed when using entrusted signed z/OSMF certificate
 
-If you do not use ZSS, this error will not happen because ZSS is on by default.
+**Symptoms**
+
+When using the entrusted signed z/OSMF certificate a problem may occur whereby the ZLUX AppServer cannot register with Eureka. Review of the logs indicate that the cause is the self-signed certificate:
+
+```
+<ZWED:198725> ZWESVUSR WARN (_zsf.bootstrap,webserver.js:156) ZWED0148E - Exception thrown when reading SAF keyring, e= Error: R_datalib call failed: function code: 01, SAF rc: 8, RACF rc: 8, RACF rsn: 44
+```
+
+**Solution**
+
+The error indicates that the keyring does not exist or cannot be found.
+
+Look at the keyring information and confirm the corresponding certificate authorities. Ensure that you specify the `certificateAuthorities` variable with the correct keyring label and the label of the conected CA in the `zowe.certificate` section of your `zowe.yaml` file. 
+
+For example, if the keyring label is `ZoweKeyring` and the LABLCERT of the connected CA is `CA Internal Cert`, the `certificateAuthorities` varaible should be `certificateAuthorities: safkeyring://ZWESVUSR/ZoweKeyring&CA Internal Cert`.
+
+## Error reported: wrong certificate setup (keyring certificate)
+
+**Symptoms**
+
+The connection failed but the certificate appears to be correct. A keyring certificate is setup which does not need a value for `password` in the `zowe.certificate.keystore.password` and `zowe.certificate.truststore.password`. 
```

Comment:
We could add more details here as well, like "The certificate appears to be correct, but the Gateway and the Discovery Service are failing to start" also it would be nice to add a real error code that is being shown, but i do not remember it